src/Devices/EthernetServer_TC.cpp

```diff
@@ -27,7 +29,8 @@ EthernetServer_TC* EthernetServer_TC::instance() {
  */
 EthernetServer_TC::EthernetServer_TC(uint16_t port) : EthernetServer(port) {
   begin();
-  serial(F("Ethernet Server is listening on port 80"));
+  IP Address = Ethernet_TC::instance()->getIP();
+  serial(F("Ethernet Server is listening on %i.%i.%i.%i:%i"), IP[0], IP[1], IP[2], IP[3], getPort());
```

Comment:
Codacy points out that the variable Address is never used. I think `IP` is the "type" and `Address` is the "name".